src/pip/_internal/__init__.py

```diff
@@ -17,7 +17,9 @@
 # to add socks as yet another dependency for pip, nor do I want to allow-stder
 # in the DEP-8 tests, so just suppress the warning.  pdb tells me this has to
 # be done before the import of pip.vcs.
-from pip._vendor.requests.packages.urllib3.exceptions import DependencyWarning
+from pip._internal.vendor.requests.packages.urllib3.exceptions import (
```

Comment:
How about keeping vendored packages in `pip._vendor`?

I know it's bike shedding but if everyone agrees it's a nicer colour, that's good too. :)

Reply:
Fine with me.